Mathlib/Algebra/Star/Unitary.lean

```diff
@@ -239,4 +240,24 @@ instance : HasDistribNeg (unitary R) :=
 
 end Ring
 
+section UnitaryConjugate
+
+universe u
+
+variable {R A : Type*} [CommSemiring R] [Ring A] [Algebra R A] [StarMul A]
+
+/-- Unitary conjugation preserves the spectrum, star on left. -/
+@[simp]
+lemma spectrum.unitary_conjugate {a : A} {u : unitary A} :
```

Comment:
Any reason for the name being the other way around for unitaries? (i.e. `unitary_conjugate` instead of `conjugate_unitary`)? I don't have a strong preference either way, but it would be nice to have the same order as for units.

Reply:
I have no good reason at all, actually. I'll coordinate these. Thanks for pointing this out!